src/core/path.c

```diff
@@ -82,9 +85,11 @@ int path_spec_watch(PathSpec *s, sd_event_io_handler_t handler) {
                         *cut = '\0';
 
                         flags = IN_MOVE_SELF | IN_DELETE_SELF | IN_CREATE | IN_MOVED_TO;
+                        SET_FLAG(flags, IN_DELETE | IN_MOVED_FROM, p && p->deactivation_toggle);
```

Comment:
hmm, i don't get this? add in IN_DELETE/IN_MOVED_FROM to the inotify mask of the parent dirs? the IN_DELETE_SELF/IN_MOVE_SELF on the contained inode we care about should be enough?